src/Functions/GatherUtils/Sources.h

```diff
@@ -755,6 +755,7 @@ struct GenericValueSource : public ValueSourceImpl<GenericValueSource>
 {
     using Slice = GenericValueSlice;
     using SinkType = GenericArraySink;
+    using Column = IColumn;
```

Comment:
@vitlibar why is that needed?

Reply:
It wouldn't be compiled without this line, because I'm using the type [ConstSource<GenericValueSource>](https://github.com/ClickHouse/ClickHouse/pull/26075/files#diff-c80e123b3b061659640ed20e8cbe00a51b326db4493e57bf2b405c0650790a38R234) and the template class `ConstSource` depends on the type [Base::Column](https://github.com/vitlibar/ClickHouse/blob/3dfdcf4604e4c13e3b90743cede89cc5ed942b5c/src/Functions/GatherUtils/Sources.h#L168).